pandas/core/base.py

```diff
@@ -952,21 +951,26 @@ def value_counts(self, normalize=False, sort=True, ascending=False,
                               normalize=normalize, bins=bins, dropna=dropna)
         return result
 
-    def unique(self):
+    _shared_docs['unique'] = (
         """
-        Return array of unique values in the object. Significantly faster than
-        numpy.unique. Includes NA values.
+        Return %(unique)s of unique values in the object.
+        Significantly faster than numpy.unique. Includes NA values.
```

Comment:
while updating. let's add that uniques are preserved in the order as seen (I know we had discussed this), but let's document the behavior at least. @jorisvandenbossche @shoyer